builtin/v16/evm/evm_state.go

```diff
@@ -9,6 +9,16 @@ import (
 	"github.com/filecoin-project/go-state-types/builtin/v16/util/adt"
 )
 
+type TransientDataLifespan struct {
+	Origin abi.ActorID
+	Nonce  uint64
+}
+
+type TransientData struct {
+	TransientDataState    cid.Cid
+	TransientDataLifespan TransientDataLifespan
+}
```

Comment:
We should try and keep the godocs consistent with the rust docs, it's very helpful to not have to go hunting for it when you're only working in Go.

```suggestion
// A structure representing Transient Data
type TransientData struct {
	TransientDataState    cid.Cid // The contract transient data state dictionary. Transient Data State is a map of U256 -> U256 values. KAMT<U256, U256>
	TransientDataLifespan TransientDataLifespan // The data representing the transient data lifespan
}
```